src/assets/src/components/common.tsx

```diff
@@ -473,3 +473,45 @@ export const Breadcrumbs = (props: BreadcrumbsProps) => {
 
     );
 }
+
+interface QueueListProps {
+    queues: ReadonlyArray<QueueBase>;
+    manageLink?: boolean | undefined;
+}
+
+export function QueueList (props: QueueListProps) {
+    const linkBase = props.manageLink ? '/manage/' : '/queue/'
+    const badgeClass = 'queue-list-badge'
+
+    const queueItems = props.queues.map(q => (
+        <Link to={`${linkBase}${q.id}`}>
+            <ListGroup.Item key={q.id}>
+                <Row>
+                    <Col lg={2} sm={2} xs={2}>
+                        <Badge className={badgeClass} variant='primary' pill={true}>{q.id}</Badge>
+                    </Col>
+                    <Col lg={8} sm={8} xs={7}>{q.name}</Col>
+                    <Col lg={2} sm={2} xs={3}>
+                        <Badge className={badgeClass} variant={q.status === 'open' ? 'success' : 'danger'} pill={true}>
+                            {q.status}
+                        </Badge>
+                    </Col>
+                </Row>
+            </ListGroup.Item>
+        </Link>
+    ));
```

Comment:
Semantically this feels more like a table than a list, but I'm fine with it as long as it causes no screen reader issues.

Reply:
Yes, I would agree. I didn't want to rock the boat too much, so I kinda just went with this. I could shift to a table if you want, probably wouldn't be too much work. My main accessibility concern with the current approach would be that the column headers aren't labeled for screen readers as such, as they would be in a table.

Reply:
Thinking more about this. We do use tables elsewhere in the application, but this feels less like presenting tabular data and more about offering a choice (of which queue to navigate to). Shifting to a table might disrupt that feeling.

In terms of helping with screen-reading given the current implementation, seems like we can use `aria-labelled-by` for each `Col` with a value and reference the appropriate header, but this seems to result in all the headers being read at once and then the values, and then (at least on Mac's VoiceOver). That could be fine?

Alternatively, we could specify `aria-label` values for each of the `Col`s, which means we could have the screenreader say the field header before the value: For example:

```
                    <Col aria-label={`ID: ${q.id}`} lg={2} sm={2} xs={2}>
                        <Badge  className={badgeClass} variant='primary' pill={true}>{q.id}</Badge>
                    </Col>
```

Reply:
@teshsopher Thoughts on reading headers first vs. repeating per value? Or any general thoughts on the semantics of table vs. list wrt accessibility?

Reply:
In this case, the "Status" data only has meaning if it's associated with the individual queue. So, the screen reader would need to tell the user something along the lines of: "[Queue Name] Status: Open"

Both a list or a table can work for this, I think. However, if it's a table, it should work like a table with two headers (column headers are "Queue Name" and "Status," and the row headers would be the individual queue name). In this case, I'm not 100% sure which would be best. I'm inclined to say table since this is a pattern we've used, but the amount of data in the table would only be one column so a list may work just as well. In either case, we would need there to be an ARIA notification to tell users what information has changed when the table updates (e.g. if two queues close while the user is on this page)

Reply:
Thanks, @teshsopher, I can draft a table version and reach out to you for feedback. I guess I was thinking there would be three columns, for each of the data elements.

With respect to ARIA notifications, is this something we are doing elsewhere in the application? I'll need to do some research on this, and I guess I wonder whether this can be made a separate issue?

Reply:
Okay, I've refactored to use a `table` approach. I will get this pushed out to `dev` for review. For the ARIA notifications on page updates, this seems important but something that should be resolved site wide and will require more research. I've created an issue for this: https://github.com/tl-its-umich-edu/remote-office-hours-queue/issues/163.